### PR TITLE
Fix Typos

### PR DESCRIPTION
**Pull request by maqnouch**

### CODE REVIEW

File: aerodoc-backend/readme.md [hunk 1]
```diff
@@ -4,12 +4,12 @@ This backend application is a showcase / quickstart part of Aerogear's UnifiedPu
 This Application shows how a business specific App can interact with [aerogear-unifiedpush-server](https://github.com/aerogear/aerogear-unifiedpush-server) . It shows how it can use the  [aerogear-unifiedpush-java-client](https://github.com/aerogear/aerogear-unifiedpush-java-client) to send Unified Messages to the push server.
 
 # Description of the Application
-
+competitve
```
Comment (edewit) [inline aerodoc-backend/readme.md]: this should not be here I'm sure ;)

**Comment by edewit** [comment]
Thanks :+1: if you remove the added misspelled word ;)

**Comment by maqnouch** [comment]
I’m sorry :(

**Comment by danielpassos** [comment]
:+1: 

@maqnouch Thank you for your PR

**Pull request merged**